demo/hello/README.md

```diff
@@ -0,0 +1,15 @@
+# Run Hello Demo
+
+```bash
+
+git clone https://github.com/dubbogo/getty.git
+
+cd getty/demo/hello
+
+# run server
+go run tcp/server/server.go
+
+# run client
+go run tcp/client/client.go
+
```

Comment:
![image](https://user-images.githubusercontent.com/18375310/61589767-965a4380-abe1-11e9-9674-65477c8c85a1.png)

As shown in the figure， I think it may be nessessary to add some instructions for `flag` in `README.md`.